gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Fri Apr 14 14:13:27 CEST 2017
+#Tue Oct 31 10:57:02 JST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-all.zip
```

Comment:
use `./gradlew wrapper --gradle-version 4.1 --distribution-type all` to upgrade gradle. You did not update the `gradle-wrapper.jar`